src/net.cpp

```diff
@@ -2063,20 +2063,15 @@ class CNetCleanup
 instance_of_cnetcleanup;
 
 
-
-
-
-
-
-void RelayTransaction(const CTransaction& tx)
+void RelayTransaction(const CTransaction& tx, CAmount fee)
```

Comment:
@MarcoFalke I wasn't sure about that.
I didn't love the idea of introducing CFeeRate into net.cpp and RelayTransaction, but if people prefer it that way I don't have a strong opinion.